src/views/tables/PositionsTable/PositionsTriggersCell.tsx

```diff
@@ -179,36 +189,43 @@ export const PositionsTriggersCell = ({
 
     return (
       <>
-        {triggerLabel({})}
+        {triggerLabel()}
         {viewOrdersButton}
       </>
     );
   };
 
   return (
-    <Styled.Cell>
+    <TableCell
+      stacked
+      slotRight={
+        !isDisabled &&
+        testFlags.isolatedMargin &&
+        complianceState === ComplianceStates.FULL_ACCESS && (
+          <Styled.EditButton
+            key="edit-margin"
+            iconName={IconName.Pencil}
+            shape={ButtonShape.Square}
+            onClick={openTriggersDialog}
+          />
```

Comment:
hmm I think this is due to how `TableCell` is rendered, but the bottom row is smaller than the top row 😢 and differently colored/styled

<img width="123" alt="Screenshot 2024-05-07 at 5 22 42 PM" src="https://github.com/dydxprotocol/v4-web/assets/70078372/c2f4e1dc-a5e7-41e3-aaf4-c5e336a99efa">

Reply:
fixed!